src/main/java/com/truelayer/java/payouts/entities/beneficiary/TransactionSearchCriteria.java

```diff
@@ -0,0 +1,27 @@
+package com.truelayer.java.payouts.entities.beneficiary;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.truelayer.java.entities.CurrencyCode;
+import java.time.LocalDate;
+import java.util.List;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+@Builder
+@Getter
+@ToString
+@EqualsAndHashCode
+public class TransactionSearchCriteria {
+    private List<String> tokens;
+
+    private Integer amountInMinor;
```

Comment:
Any reason why we're using `Integer` instead of `int` here? Not a blocker, just a question related to consistency as I can see we're using `int` for the same property in other places